src/components/CategorySort.jsx

```diff
@@ -19,7 +19,7 @@ const CategorySort = ({setSelectedCategory, selectedCategory}) => {
     }
     }
 
-    useEffect(() => {
+    useEffect((setErrorMessage) => {
         getCategories().then((categories) => {
```

Comment:
Hey Lily, we don't appear to be getting an error message displayed on page when looking for a topic that does not exist. We want to do the same here as we did with the article id if it does not exist. 🙂